plugins/uartdmx/UartDmxDevice.h

```diff
@@ -60,11 +61,14 @@ class UartDmxDevice : public Device {
   const std::string m_path;
   unsigned int m_breakt;
   unsigned int m_malft;
+  unsigned int m_paddingt;
```

Comment:
I think the t's were timing, e..g break time, mark after last frame time, so this should probably just be m_padding.